README.md

```diff
@@ -1,5 +1,9 @@
 # Lumen GraphQL
 
+[![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](https://raw.githubusercontent.com/digiaonline/lumen-graphql/master/LICENSE)
+[![Total Downloads](https://poser.pugx.org/digiaonline/lumen-graphql/downloads)](https://packagist.org/packages/digiaonline/lumen-graphql)
+[![Latest Stable Version](https://poser.pugx.org/digiaonline/lumen-graphql/v/stable)](https://packagist.org/packages/digiaonline/lumen-graphql)
```

Comment:
Let's follow the same order as other projects, for example:
https://github.com/digiaonline/lumen-elasticsearch

Reply:
Done.